oci-unit-tests/standalone-checks_test.sh

```diff
@@ -93,9 +105,119 @@ test_all_supported_architectures_are_available()
         done
     done
 
-    rm -rf "${utilsdir}"
-
     assertTrue "Not all supported architectures are available" "$ret"
 }
 
+test_images_hashes_are_equal()
+{
+    # We currently support only the "ubuntu" namespace, because it's
+    # the only one that is published on both registries.
+    local namespaces="ubuntu"
+    local images_list
+    local ret=0
+
+    if [ -z "${DOCKER_USERNAME}" ] || [ -z "${DOCKER_PASSWORD}" ]; then
```

Comment:
Unfortunately they need to be set because the scripts in the `utils` directory don't accept anonymous queries (which is kind of a good thing, because non-anonymous queries have a higher API ratelimit).